client/app/vps/upgrade/vps-upgrade.less

```diff
@@ -0,0 +1,13 @@
+.vps-upgrade {
+  .oui-collapsible__body {
+    overflow-y: auto;
+    max-height: 300px;
+
+    pre {
```

Comment:
Could be also added to https://github.com/ovh-ux/ovh-ui-kit/blob/master/packages/oui-collapsible/collapsible.less#L38 or it's a specific case?

Reply:
Mmmmh. This is a specific case for contracts.
I think it should be better to update https://github.com/ovh-ux/ng-ovh-contracts instead.